src/Template/GoldenEye.WebApi.SimpleDDD/Startup.cs

```diff
@@ -0,0 +1,68 @@
+﻿using AutoMapper;
+using GoldenEye.Backend.Core.DDD.Registration;
+using GoldenEye.Backend.Core.WebApi.Modules;
+using GoldenEye.Backend.Core.WebApi.Registration;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Backend.DDD.WebApi.Sample
+{
+    public class Startup
```

Comment:
See new improved `Startup.cs` in https://github.com/oskardudycz/GoldenEye/pull/58/files#diff-49cb8c2ec8c71b61bd15b221102c7dd6